src/main/java/com/zufar/icedlatte/order/endpoint/OrderEndpoint.java

```diff
@@ -0,0 +1,45 @@
+package com.zufar.icedlatte.order.endpoint;
+
+import com.zufar.icedlatte.openapi.dto.AddedOrder;
+import com.zufar.icedlatte.openapi.dto.OrderDto;
+import com.zufar.icedlatte.openapi.dto.OrderStatus;
+import com.zufar.icedlatte.order.api.OrderApi;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@Validated
+@RequestMapping(value = OrderEndpoint.ORDER_URL)
+public class OrderEndpoint implements com.zufar.icedlatte.openapi.order.api.OrdersApi{
+
+    public static final String ORDER_URL = "/api/v1/orders";
+
+    private final OrderApi orderApi;
+
+    @Override
+    @PostMapping
+    public ResponseEntity<AddedOrder> addOrder(final OrderDto request) {
+        log.info("Received request to add order.");
+        var addedOrder = orderApi.addNewOrder(request);
+        log.info("Order was added with id: {}", addedOrder.getId());
+        return ResponseEntity.ok().body(addedOrder);
+    }
+
+    @Override
+    @GetMapping
+    public ResponseEntity<List<AddedOrder>> getListOfOrdersForUser(final List<OrderStatus> statusList) {
```

Comment:
getOrders is enough

Reply:
Renamed in [46e65b4](https://github.com/Sunagatov/Iced-Latte/pull/217/commits/46e65b4c9646349a131383ace42afdeddba4b663)